docs/api.rst

```diff
@@ -1,23 +1,203 @@
+      - ``c`` and ``cpp``
+
+   .. admonition:: Example
+      :class: example
+
+      To use the first sentence rather than the first paragraph as the synopsis
+      for C++ class descriptions, add the following to :file:`conf.py`:
+
+      .. code-block:: python
+
+         object_description_options_map = [
+             ("cpp:class", dict(generate_synopses="first_sentence")),
+         ]
+
+.. objconf:: include_object_type_in_xref_tooltip
+
+   `bool` value that indicates whether to include the object type in
+   cross-reference and TOC tooltips.  Defaults to :python:`True`.
+
+   .. note::
+
+      For TOC entries, this is supported for all domains.  For regular cross
+      references, this is supported only for the following domains:
+
+      - ``std`` (including object types added using :py:obj:`sphinx.application.Sphinx.add_object_type`)
+      - ``c`` and ``cpp``
+
+   .. admonition:: Example
+      :class: example
+
+      To exclude the object type from all ``py`` domain xrefs, add the following
+      to :file:`conf.py`:
+
+      .. code-block:: python
+
+         object_description_options_map = [
+             ("py:.*", dict(include_object_type_in_xref_tooltip=False)),
+         ]
+
+.. objconf:: include_object_description_fields_in_toc
+
+   :python:`bool` indicating whether to include fields, like "Parameters",
+   "Returns", "Raises", etc. in the table of contents.  Defaults to
+   :python:`True`.
 
    For an example, see: :cpp:expr:`synopses_ex::Foo` and note the ``Template
    Parameters``, ``Parameters``, and ``Returns`` headings shown in the
    right-side table of contents.
 
    .. note::
 
-      This option does not control whether there are separate TOC entries for
-      individual parameters, such as for :cpp:expr:`synopses_ex::Foo::T`,
+      To control whether there are separate TOC entries for individual
+      parameters, such as for :cpp:expr:`synopses_ex::Foo::T`,
       :cpp:expr:`synopses_ex::Foo::N`, :cpp:expr:`synopses_ex::Foo::param`, and
-      :cpp:expr:`synopses_ex::Foo::arr`.  Currently, for the C and C++ domains,
-      any parameter documented by a :rst:``:param x:`` field will always result
-      in a TOC entry, regardless of the value of
-      :confval:`include_object_description_fields_in_toc`.  Other domains are
-      not yet supported.
+      :cpp:expr:`synopses_ex::Foo::arr`, use the :objconf:`include_in_toc`
+      option.
+
+
+   .. admonition:: Example
+      :class: example
+
+      To exclude object description fields from the table of contents for all
+      ``py`` domain objects, add the following to :file:`conf.py`:
+
+      .. code-block:: python
+
+         object_description_options_map = [
+             ("py:.*", dict(include_object_description_fields_in_toc=False)),
+         ]
+
+Other options described elsewhere include:
+
+- :objconf:`html_wrap_signatures_with_css`
+- :objconf:`html_wrap_signatures_with_css_column_limit`
```

Comment:
```suggestion
- :objconf:`wrap_signatures_with_css`
- :objconf:`wrap_signatures_with_css_column_limit`
```
should fix the resolution of the pending refs
![image](https://user-images.githubusercontent.com/14963867/165223705-d240877f-d658-4819-a057-88f21170e783.png)